packages/grid/x-data-grid/src/tests/quickFiltering.DataGrid.test.tsx

```diff
@@ -179,6 +179,46 @@ describe('<DataGrid /> - Quick Filter', () => {
       clock.runToLast();
       expect(getColumnValues(0)).to.deep.equal(['Nike', 'Adidas']);
     });
+
```

Comment:
Although it's not the core purpose to use it via controlled prop, we couldn't stop users from doing so, since it's part of `filterModel`. Do you think that also adding a small test that reacts to the controlled model change in perspective of this prop and recomputes the filter results will make sense?

Reply:
Test added, thanks for your suggestions!

Reply:
Thanks, looks great!